### PR TITLE
Docs for some JS views

### PR DESCRIPTION
**Pull request by danielbachhuber**
See #16

### CODE REVIEW

**Comment by danielbachhuber** [comment]
@goldenapples #reviewmerge

File: js/src/controllers/media-controller.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ var MediaController = wp.media.controller.State.extend({
 		if ( this.frame && this.frame.toolbar ) {
 			this.frame.toolbar.get().refresh();
 		}
+		this.destroySelect2UI();
```
Comment (goldenapples) [inline js/src/controllers/media-controller.js:24]: These changes look unrelated to the documentation changes.

File: js/src/views/edit-attribute-field-attachment.js [hunk 2]
```diff
@@ -10,11 +18,10 @@ var editAttributeFieldAttachment = sui.views.editAttributeField.extend( {
...
 
 	/**
-	 * Update the field attachment.
-	 * Re-renders UI.
-	 * If ID is empty - does nothing.
+	 * Update the Field Attachment to use a new attachment ID.
+	 * Re-renders UI. But, bails early if the ID isn't set.
 	 *
-	 * @param {int} id Attachment ID
+	 * @param integer id Attachment ID
```
Comment (goldenapples) [inline js/src/views/edit-attribute-field-attachment.js:24]: What is the distinction between when a type description is shown in braces and not? Should this be `{integer}`?

File: js/src/views/edit-attribute-field-attachment.js [hunk 3]
```diff
@@ -132,6 +142,9 @@ var editAttributeFieldAttachment = sui.views.editAttributeField.extend( {
 	/**
 	 * Open media frame when add button is clicked.
 	 *
+	 * Sets the frame to display the media library with the attachment selected if set.
+	 *
+	 * @param {object} e
```
Comment (goldenapples) [inline js/src/views/edit-attribute-field-attachment.js:147]: I'd call this param an `{Event}` for clarity.

File: js/src/views/edit-attribute-field-attachment.js [hunk 4]
```diff
@@ -1,6 +1,14 @@
 var sui = require('sui-utils/sui');
 
-var editAttributeFieldAttachment = sui.views.editAttributeField.extend( {
+/**
+ * sui.views.editAttributeFieldAttachment
+ *
+ * The controller for the Attachment Field.
```
Comment (goldenapples) [inline js/src/views/edit-attribute-field-attachment.js:6]: More accurately, this is a View class, not a Controller.

File: js/src/views/edit-attribute-field-attachment.js [hunk 5]
```diff
@@ -154,6 +167,7 @@ var editAttributeFieldAttachment = sui.views.editAttributeField.extend( {
 	/**
 	 * When an attachment is selected from the media frame, update the model value.
 	 *
+	 * @param {object} e
```
Comment (goldenapples) [inline js/src/views/edit-attribute-field-attachment.js:170]: Also an `{Event}`

File: js/src/views/edit-attribute-field-attachment.js [hunk 6]
```diff
@@ -170,6 +184,8 @@ var editAttributeFieldAttachment = sui.views.editAttributeField.extend( {
 	/**
 	 * Remove the attachment.
 	 * Render preview & Update the model.
+	 *
+	 * @param {object} e
```
Comment (goldenapples) [inline js/src/views/edit-attribute-field-attachment.js:188]: Here too.

File: js/src/views/edit-attribute-field-color.js [hunk 7]
```diff
@@ -2,12 +2,23 @@ var sui = require('sui-utils/sui'),
     editAttributeField = require( 'sui-views/edit-attribute-field' ),
     $ = require('jquery');
 
+/**
+ * sui.views.editAttributeFieldColor
+ *
+ * The controller for the Color Field.
```
Comment (goldenapples) [inline js/src/views/edit-attribute-field-color.js:8]: View, not controlller?

**Comment by goldenapples** [comment]
Hmm. I'll read through this some more and read up on jsdoc rules. In the meantime, I left a few comments that jumped out at me.

**Comment by danielbachhuber** [comment]
Going to attempt this again...
